.github/workflows/ci.yml

```diff
@@ -31,3 +31,18 @@ jobs:
     - name: Run other tests
       run: |
         dotnet test
+
+  build_with_fsharp_from_mono:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: |
+        DEBIAN_FRONTEND=noninteractive sudo apt install -y msbuild fsharp nuget
+
+        # override TargetFrameworks to not build netstandard2.1 (incompatible with Mono)
+        sed -i "7i<PropertyGroup><TargetFrameworks>netstandard2.0</TargetFrameworks></PropertyGroup>" src/DotNetLightning.Core/DotNetLightning.Core.fsproj
```

Comment:
Is this necessary?  Is it impossible to specify the target framework from the command line argument for msbuild?

Reply:
I guess not, but I'll double check

Reply:
you were right; fixed